docs/reference/cluster/stats.asciidoc

```diff
@@ -1307,6 +1307,142 @@ Each repository type may also include other statistics about the repositories of
 
 ====
 
+`ccs`::
+(object) Contains information relating to <<modules-cross-cluster-search, cross-cluster search>> settings and activity in the cluster.
++
+.Properties of `ccs`
+[%collapsible%open]
+=====
+
+
+`_search`:::
+(object) Contains telemetry information about the <<modules-cross-cluster-search, cross-cluster searches>> in the cluster.
++
+.Properties of `_search`
+[%collapsible%open]
+======
+`total`:::
+(integer) The total number of cross-cluster search requests that have been executed by the cluster.
+
+`success`:::
+(integer) The total number of cross-cluster search requests that have been successfully executed by the cluster.
```

Comment:
Is "success" inclusive of "skipped"? Might be good to indicate that.

Reply:
In this particular case, it may be either. This is for the whole request, and the request may be successful even if it included skipped clusters, but it also may fail for some reason. I think `skipped` counts only successful requests that had skipped clusters, but I'll check and add clarification.